sim/druid/eclipse.go

```diff
@@ -56,15 +56,29 @@ func (druid *Druid) EnableEclipseBar() {
 }
 
 func (druid *Druid) RegisterEclipseAuras() {
+	masteryBonus := 0.16 + druid.GetMasteryPoints()*0.02
```

Comment:
You dont have mastery fully initialized during registration of spells/auras. You need to update the value again in the auras OnGain methods before activating the mods. Also you should register to the On Mastery Changed callback to update the spell mods values when mastery changes due to procs

Reply:
I can move it there, but I don't think I need to update it as Eclipse is supposedly snapshotting mastery.

Reply:
the spell mod should still be properly updated on mastery changes. The dot will still properly snapshot the damage on application but you still have to have the correct value to snapshot from.

Reply:
yep, that's done.